<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialBox5_lite</class>
 <widget class="QWidget" name="SerialBox5_lite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>72</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_power">
        <property name="maximumSize">
         <size>
          <width>33</width>
          <height>70</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QComboBox" name="cb_BaudBox"/>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="btn_refresh">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="cb_PortBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_power"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_right_LOG"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_SEND"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_PortBox</tabstop>
  <tabstop>btn_refresh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
